DllImportGenerator/DllImportGenerator.UnitTests/Compiles.cs

```diff
@@ -91,6 +91,11 @@ public static IEnumerable<object[]> CodeSnippetsToCompile_NoDiagnostics()
             yield return new[] { CodeSnippets.ArrayParameterWithNestedMarshalInfo<string>(UnmanagedType.LPUTF8Str) };
             yield return new[] { CodeSnippets.ArrayParameterWithNestedMarshalInfo<string>(UnmanagedType.LPStr) };
 
+            // [In, Out] attributes
+            // By value non-blittable array
+            yield return new[] { CodeSnippets.ByValueParameterWithModifier<bool[]>("Out") };
+            yield return new[] { CodeSnippets.ByValueParameterWithModifier<bool[]>("In, Out") };
```

Comment:
Seems like we are missing non-blittable with just `In`. I think you mentioned this previously as an issue with detection, is that true?

Reply:
Non-blittable with just `In` is unsupported. We only support `[Out]` and `[In, Out]` since `[In]` on its own doesn't do anything. Looks like I forgot to add a test for this in CompileFails though, so I'll be sure to add that. This is not an issue with detection; that issue I was seeing was due to a bug.